marko/renderer.py

```diff
@@ -82,18 +81,11 @@ def render_children(self, element):  # type: (Element) -> str
         rendered = [self.render(child) for child in element.children]  # type: ignore
         return "".join(rendered)
 
-    def _cls_to_func_name(self, klass):  # type: (ElementType) -> str
-        from .block import parser
-
-        element_types = itertools.chain(
-            parser.block_elements.items(),  # type: ignore
-            parser.inline_elements.items(),  # type: ignore
-        )
-        for name, cls in element_types:
-            if cls is klass:
-                return "render_" + camel_to_snake_case(name)
-
-        return "render_children"
+    def _el_to_func_name(self, element):  # type: (Union[Element, List[T]]) -> str
+        try:
+            return "render_" + element.get_element_type(snake_case=True)
```

Comment:
I wonder if the typechecker would be happy on this.

Reply:
I just tested this, and indeed, it fails static typechecking